include/CMakeLists.txt

```
@@ -1,10 +1,44 @@
 option(INSTALL_TF_PSA_CRYPTO_HEADERS "Install TF PSA Crypto headers." ON)
 
+# The handling of TF_PSA_CRYPTO_CONFIG_NAME assumes that the default
+# configuration file 'include/psa/crypto_config.h' exists in the source tree.
+# That file is copied into the build tree and then modified with 'config.py'
+# according to the configuration name provided via TF_PSA_CRYPTO_CONFIG_NAME.
+set(TF_PSA_CRYPTO_CONFIG_NAME "" CACHE STRING "Name of the configuration to use (see config.py documentation)")
+
+if(TF_PSA_CRYPTO_CONFIG_NAME)
```

Minor: this would take the wrong branch if we had a config name that [cmake](https://cmake.org/cmake/help/v3.5/command/if.html) considers to be a false constant, such as `off` or `file-notfound`. Same with the config file name (fortunately no false constant ends with `.h`). I think this should be

```suggestion
if(NOT "${TF_PSA_CRYPTO_CONFIG_NAME}" STREQUAL "")
```